### ISSUE TITLE
Provide support for defining ruleset filepath

### ISSUE THREAD

**Issue by mikeerickson** [bpearson/atom-phpcs#27]
Hey Gang

Just started with this plugin and all is working now (as expected).   

But I wasted a good hour trying to debug this why it wouldn't work out of the box

In short, our ruleset name was `phpcs-ruleset.xml` (see hyphen)   The plugin is searching for `phpcs.ruleset.xml`   

So, while it was an easy fix on our end to rename ruleset, we have a large organization of folks who use this company ruleset.   Most of the team uses PHPStorm, where you can provide a direct path to .xml file.

In short, if I had a way to supply path to ruleset, we don't have to change anything in our repos and we wont get out of sync on changes to ruleset etc.

**Comment by bpearson** [comment]
I have tried a custom ruleset by adding the full path in the standard setting in preferences, and it worked for me.

**Comment by mikeerickson** [comment]
Hmm, I looked all over settings and only thing I saw regarding ruleset was the checkbox which stated it would search for results 

I will have a look again tomorrow when I return to work where I use plugin

**Comment by bpearson** [comment]
Hmm? bit puzzled by what you are seeing... I get this... 
![screenshot_2016-12-20_11-51-18](https://cloud.githubusercontent.com/assets/997686/21335007/8e17d9a4-c6af-11e6-9f40-93bb15910c41.png)
And put the full path in the "Standard"

**Comment by mikeerickson** [comment]
Ah, didn't think about using the standard property, but I do see that.  I will try It out in the morning

**Comment by mikeerickson** [comment]
OK, all is good.  I have put the name of our desired ruleset into this property and all is good.
